test/Microsoft.Dnx.Testing.SampleTests/DnuCommonUtilTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.Dnx.Testing.SampleTests
+{
+    [Collection(nameof(SampleTestCollection))]
+    public class DnuCommonUtilTests : DnxSdkFunctionalTestBase
+    {
+        [Theory, TraceTest]
+        [MemberData(nameof(DnxSdks))]
+        public void FailTest(DnxSdk sdk)
+        {
+            var feedSolution = TestUtils.GetSolution<DnuCommonUtilTests>(sdk, "DependencyGraphsProject");
```

Comment:
The using pattern is cleaner and makes it so that user code doesn't have to write try {}, finally{}